Package.swift

```diff
@@ -11,35 +20,85 @@ let package = Package(
         .package(url: "https://github.com/nodes-vapor/keychain.git", from: "2.0.0-rc"),
         .package(url: "https://github.com/nodes-vapor/submissions.git", from: "3.0.0-rc"),
         .package(url: "https://github.com/vapor-community/mailgun.git", from: "5.0.0"),
+        .package(url: "https://github.com/nodes-vapor/n-meta.git", from: "4.0.0-rc"),
         .package(url: "https://github.com/vapor/fluent-postgres-driver.git", from: "2.0.0"),
         .package(url: "https://github.com/vapor/fluent.git", from: "4.0.0"),
         .package(url: "https://github.com/vapor/leaf.git", from: "4.0.0"),
         .package(url: "https://github.com/vapor/vapor.git", from: "4.0.0")
     ],
     targets: [
+        .executableTarget(
+            name: "Run",
+            dependencies: [
+                "App",
+            ]
+        ),
         .target(
             name: "App",
             dependencies: [
+                "Auth",
+                "Core",
+                "User",
                 .product(name: "Bugsnag", package: "bugsnag"),
-                .product(name: "Fluent", package: "fluent"),
-                .product(name: "FluentPostgresDriver", package: "fluent-postgres-driver"),
+                .product(name: "Fluent", package: "fluent"), // remove
+                .product(name: "FluentPostgresDriver", package: "fluent-postgres-driver"), // remove
```

Comment:
Will they be removed in another pr?

Reply:
good catch! completely forgot to take those out 😅